frontend/src/config/cambodia/layers.json

```diff
@@ -1950,7 +1950,13 @@
   "cdi": {
     "title": "Combined Drought Index",
     "type": "composite",
-    "base_url": "https://to-fill-later",
+    "scale": "monthly",
```

Comment:
The term `scale` feels off to me but not sure what would be better. `period`? or `interval`? Is that a term coming from HIP? Or a standardized notion?

Reply:
Good call, I like period better. I'll update.

This is all internal to prism and it not used directly by hip-analysis or hip-service